azure-ipam/ipam.go

```diff
@@ -85,17 +106,25 @@ func (p *IPAMPlugin) CmdAdd(args *cniSkel.CmdArgs) error {
 		p.logger.Error("Failed to interpret CNS IPConfigResponse", zap.Error(err), zap.Any("response", resp))
 		return cniTypes.NewError(ErrProcessIPConfigResponse, err.Error(), "failed to interpret CNS IPConfigResponse")
 	}
-	p.logger.Debug("Parsed pod IP", zap.String("podIPNet", podIPNet.String()))
-
-	cniResult := &types100.Result{
-		IPs: []*types100.IPConfig{
-			{
-				Address: net.IPNet{
-					IP:   net.ParseIP(podIPNet.Addr().String()),
-					Mask: net.CIDRMask(podIPNet.Bits(), 32), // nolint
-				},
-			},
-		},
+	for _, IPNet := range *podIPNet {
+		p.logger.Debug("Parsed pod IP", zap.String("podIPNet", IPNet.String()))
+	}
+
+	cniResult := &types100.Result{}
+	for _, IPNet := range *podIPNet {
```

Comment:
```suggestion
	for _, ipNet := range *podIPNet {
```
I don't care as much what this variable is named as long as it is sane, but we shouldn't export it (aka shouldn't capitalize it)

Reply:
I never made the exact connection that capitalization was reserved for variables being exported from another package. I'll make sure to keep this in mind and updated it.